templates/items/parts/item-description2.hbs

```diff
@@ -0,0 +1,95 @@
+{{#*inline "description"}}
+    <div class="card description {{#if collapsible}}collapsible{{/if}} {{#if (lookup @root.collapsed target)}}collapsed{{/if}}"
+         data-target="{{ target }}">
+        <div class="header">
+            <span>{{ localize label }}</span>
+            {{#if @root.owner}}
+            <button type="button" class="description-edit unbutton" data-target="{{ target }}"
+                    aria-label="{{ localize "DND5E.DescriptionEdit" description=(localize label) }}">
+                <i class="fas fa-feather"></i>
+            </button>
+            {{/if}}
+        </div>
+        <div class="details collapsible-content">
+            <div class="wrapper">
+                {{ editor enriched target=target button=false editable=false engine="prosemirror" }}
```

Comment:
Right now nothing appears to happen when collapsing/expanding a section with no description. Perhaps some placeholder text that indicates no description is present would help.

Reply:
I've removed the suggestion of interactivity on these empty sections for now, but we can change it if ends up being a bit confusing (it might).